src/Assets/updateScreen.js

```diff
@@ -36,8 +36,10 @@ const getScreensBody = (formData, languageCode) => {
     has_mydenver: formData.benefits.mydenver,
     has_nslp: formData.benefits.nslp,
     has_oap: formData.benefits.oap,
+    has_pell: formData.benefits.pell,
```

Comment:
I am going to use `has_pell_grant` for the API.

Reply:
I've updated the API value to use `has_pell_grant`.